### DISCUSSION TITLE
What's good?

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#11] [Feedback]
### Please share what you _like_ about JG RTX
Example topics:
- What is appealing about JG RTX?
- Which textures are satisfactory and do not need to be updated or changed?

**Comment by erich666** [comment]
What's appealing? For me, a few things: nice PBR textures, but not insanely high-resolution (who honestly needs 2048x2048 - that's like one block filling the screen; 512x512 is like 4 blocks); lots of textures - I count 1324 that I use, as of 0.14.2; Creative Commons, vs. annoying Patreon firewalling; responsive creator :)

Textures: most look pretty good to me, though I haven't carefully examined each one. Easier is "which might be worth updating?" as that's a much shorter list. Cats (though barrier.png should stay :) ), redstone torch off looks white (reported elsewhere) - I'd prefer dull red, cake seems simplistic (and there seems to be an in-joke with the top), red sandstone top seems a different scale of bumpiness than the sides, stripped acacia top doesn't look great around the edges, slime is red not green (I'm a traditionalist), that's about all I notice. The rest I think are pretty great, from what I've examined so far (and there's so much to look at...).

Oh, and please do eventually add a nether quartz ore and dried kelp.

Anyway, you asked. Honestly, I'm thrilled you've done so much and keep on adding to it.

A shot from a portion of the Imperial Block Testing Farm with 0.14.2 (so you may already have made changes) - red sandstone highlighted:

![capture 2021-03-14 20 07 06](https://user-images.githubusercontent.com/803995/111089165-07f6ac00-8501-11eb-9a84-269c285999d9.jpg)

**Comment by art926** [comment]
I like that most blocks (or maybe even all of them?) seems to be implemented in the pack, in contrast to other packs. I like that material glossiness is not abused (other packs sometimes make everything look like a wet plastic). Normal maps are cool and add realism. Colors in general are not over saturated, quite realistic.

**Comment by justincrapse** [comment]
This is incredible work! I've been working on my RTX texture pack a lot this year and now understand how much goes into this. It is a LOT. 
Some/a lot of this is stunning. I don't even want to make doors now; I'm just going to use yours (not in my texture pack of course, but to supplement during my play.) 

I haven't shared this yet, but I wrote a little python framework that automatically takes the textures and outputs them correctly into the specified resource pack. So I can re-output at any resolution at the touch of a button. Let me know if you are interested.

**Reply by justincrapse** [reply]
Blocks second to none:
* All the doors and trap doors
* most of the glass
* gold and iron blocks
* Emerald block
* diamond block
* lapiz block
* logs and stiped logs
* cacti
* barrel
* smoker
* smithing table
* anvil
* campfire
* bee nest/box
* crafting trable
* dead coral (one of them done)
* cartographer table
* archer block
* redstone lantern 
* music boxes (awesome)
* end portal 
* redstone block (during the day, not a fan of it in the dark)
* observer
* tnt
* loom
* best purple portal pattern ever

I'd say these blocks are all well designed, look great, and I'd love to see them in my world. Incredible work.

**Reply by jasonjgardner** [reply]
Thanks for the feedback and sorry for the late response!

This project has a Creative Commons license. You're welcome to reuse the door textures in your pack if it has a similar license. I plan on redoing the door textures anyway. I've learned a lot since making them and can do a much better job on the next attempt.

Yeah, I'm interested in trying your script! Though I still might end up writing my own in JavaScript. I'm working on creating packs programmatically with NodeJS and [sharp](https://github.com/lovell/sharp), which includes the ability to output in multiple resolutions. I don't know Python, but I'm curious about any and all Minecraft dev tools, scripts, etc.

**Reply by justincrapse** [reply]
@jasonjgardner exactly, I can point to source textures per specified block, automatically convert those to any resolution, combine the mer textures into one file, and spit it out to a specified location. Unfortunately MER files have to be PNG (bleeding artifact issue with compression), but I save a lot of memory with the normals and basecolor files with jpeg (still looks sharp.) 

I think I'm ready to put my pack out there similar to how you are doing it. I'd love to share my tool as well. I've been trying to make it as simple as possible to use and configure and I think it is almost there.
